src/v/features/feature_table.h

```diff
@@ -720,6 +735,14 @@ class feature_table {
     // Currently loaded redpanda license details
     std::optional<security::license> _license;
 
+    // Built in trial license to fall back to if there is no license set
+    std::optional<security::license> _builtin_trial_license;
+
+    // On startup the `_builtin_trial_license` is not available until the
+    // cluster is created. Until then, we fall back to this permissive
+    // bootstrap license.
+    security::license _bootstrap_builtin_trial_license;
+
```

Comment:
I wonder if there's a way to only need to hold one copy of the license.  Would need to ensure that the built-in being set via `set_builtin_trial_license` can't override the true license.

Would also maybe need a flag indicating whether or not to return a license from the `get_license` endpoint if it's the built in eval vs the other two.

Reply:
I've added the `should_sanction` method now which avoids the complexity of `_bootstrap_builtin_trial_license `